config-templates/redhat/ocs-local/4.6/custom-parameters.json

```
@@ -0,0 +1,51 @@
+[
+  {
+    "description": "Cluster Name of OCS",
+    "displayname": "Cluster Name",
+    "name": "ocs-cluster-name",
+    "required": "true"
+  },
+  {
+    "description": "Enter the OSD device path. Mandatory",
+    "displayname": "Osd Device Path",
+    "name": "osd-device-path",
+    "required": "true",
+    "type": "csv"
+  },
+  {
+    "description": "Enter the MON device path. Mandatory",
```

Instead of `device-path` should the OSD & MON values be "by-id"?
```suggestion
    "description": "Required. Enter the MON device path.",
```